pkgs/development/python-modules/git-filter-repo/default.nix

```
@@ -0,0 +1,21 @@
+ { lib, buildPythonPackage, fetchPypi, setuptools_scm, click }:
+ buildPythonPackage rec {
+   pname = "git-filter-repo";
+   version = "2.28.0";
+
+   src = fetchPypi {
+     inherit pname version;
+     sha256 = "0zgvbyvj3i1k9dkrzhw8r9jrylyb0454ysac8209lj46hcmzfdcv";
+   };
+
+   propagatedBuildInputs = [
+     click
+     setuptools_scm
+   ];
```

`setuptools_scm` is only needed at build time
```suggestion
   nativeBuildInputs = [
     setuptools_scm
   ];
   
   propagatedBuildInputs = [
     click
   ];
```